api/src/domain/mutations/add-organizations-domains.js

```diff
@@ -0,0 +1,356 @@
+import { GraphQLNonNull, GraphQLList, GraphQLID, GraphQLBoolean } from 'graphql'
+import { mutationWithClientMutationId, fromGlobalId } from 'graphql-relay'
+import { t } from '@lingui/macro'
+
+import { removeDomainUnion } from '../unions'
+import { Domain } from '../../scalars'
+import { logActivity } from '../../audit-logs/mutations/log-activity'
+
+export const addOrganizationsDomains = new mutationWithClientMutationId({
+  name: 'AddOrganizationsDomains',
+  description: 'Mutation used to create a new domain for an organization.',
+  inputFields: () => ({
+    orgId: {
+      type: GraphQLNonNull(GraphQLID),
+      description:
+        'The global id of the organization you wish to assign this domain to.',
+    },
+    domains: {
+      type: GraphQLNonNull(new GraphQLList(Domain)),
+      description: 'Url that you would like to be added to the database.',
+    },
+    hideNewDomains: {
+      type: GraphQLBoolean,
+      description: 'New domains will be hidden.',
+    },
+    tagNewDomains: {
+      type: GraphQLBoolean,
+      description: 'New domains will be tagged with NEW.',
+    },
+    audit: {
+      type: GraphQLBoolean,
+      description: 'Audit logs will be created.',
+    },
+  }),
+  outputFields: () => ({
+    result: {
+      type: removeDomainUnion,
+      description:
+        '`RemoveDomainUnion` returning either a `DomainResultType`, or `DomainErrorType` object.',
+      resolve: (payload) => payload,
```

Comment:
Wrong return type/ domains not returned for this mutation. Asking for domain info in the gql query returns `null`:

```
mutation {
  addOrganizationsDomains(input:{orgId: "ordId", domains: [
    "example1234.com"
  ]}) {
    result {
      ... on DomainResult {
        status
        domain {
          domain
        }
      }
    }
  }
}
```

returns 

```
{
  "data": {
    "addOrganizationsDomains": {
      "result": {
        "status": "Successfully added 0 domain(s) to some-org.",
        "domain": null
      }
    }
  }
}
```